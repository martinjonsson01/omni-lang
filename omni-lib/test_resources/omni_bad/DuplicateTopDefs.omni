module DuplicateTopDefs()

function() : Unit = ()

function() : Unit = ()

main() : Int = 0