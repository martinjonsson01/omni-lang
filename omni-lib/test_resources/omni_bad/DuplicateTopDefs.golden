error:
duplicate identifier 'function'
at
test_resources\omni_bad\DuplicateTopDefs.omni:3:1-3:9
test_resources\omni_bad\DuplicateTopDefs.omni:5:1-5:9
     